common/Environments/Helpers/ComputeSystemHelpers.cs

```diff
@@ -126,4 +129,19 @@ public static EnvironmentsCallToActionData UpdateCallToActionText(int providerCo
 
         return new(navigateToExtensionsLibrary, callToActionText, callToActionHyperLinkText);
     }
+
+    public static void RemoveAllItems<T>(ObservableCollection<T> collection)

```

Comment:
Could you add a comment why we have this method instead of Clear(). In time, nobody will remember PR's description.

Reply:
Sure will do